saleor/payment/gateways/stripe/webhooks.py

```diff
@@ -217,6 +218,19 @@ def handle_authorized_payment_intent(
             extra={"payment_intent": payment_intent.id},
         )
         return
+
+    if not payment.is_active:
+        transaction = _update_payment_with_new_transaction(
+            payment,
+            payment_intent,
+            TransactionKind.AUTH,
+            payment_intent.amount,
+            payment_intent.currency,
+        )
+        manager = get_plugins_manager()
```

Comment:
In PR #8544 this function is refactored to receive plugins manager as an argument; it also changes the logic here: https://github.com/saleor/saleor/pull/8544/files#diff-2213dac925da7e225ab80a2e22732c5bfd480e7cf86072990f990409d3b1ef70R190

Reply:
Agreed but It's refactored on the feature branch (partial payments) but we need it now on `3.0`.

Reply:
I merged this PR into partial payments here: https://github.com/SaleorIntegrations/saleor/commit/d6cd634721e456e64fe0619abe8cec2e967478a7. They are on their way to 3.0 next week so I think it would be good to test those 2 features together